packages/apps/plugins/plugin-markdown/src/components/EditorMain.tsx

```diff
@@ -75,23 +69,21 @@ const EditorMain = ({ model, comments, toolbar, extensions: _extensions, ...prop
     <>
       {toolbar && (
         <Toolbar.Root
-          onAction={handleAction}
-          state={formattingState}
           classNames='max-is-[60rem] justify-self-center border-be separator-separator'
+          state={formattingState}
+          onAction={handleAction}
         >
           <Toolbar.Markdown />
           <Toolbar.Separator />
           <Toolbar.Extended />
         </Toolbar.Root>
       )}
-      <div
-        role='none'
-        data-toolbar={toolbar ? 'enabled' : 'disabled'}
-        className='is-full bs-full overflow-hidden data-[toolbar=disabled]:pbs-8'
```

Comment:
Why did you remove `data-[toolbar=disabled]:pbs-8`? This was added to address a complaint that the editor was too close to the top of the viewport when the toolbar is disabled.

If it is more valuable not to do that (I would encourage checking what it looks like with the toolbar disabled), then please also remove the `data-toolbar` attribute which is no longer necessary.

Reply:
my guess is the feedback was since previously the text area padding wasn't working since  the `!md:px-8` was `md:!px-8` and not working?

Reply:
No, I can find the issue if you like, it was because without the toolbar the top edge of the “paper” effect of the document was flush with the bottom edge of the topbar.

If this wasn’t removed to fix something else, I think it’s safest to restore it.

Reply:
ah ok